source/includes/sample-write-application.java

```diff
@@ -0,0 +1,47 @@
+import com.mongodb.*;
```

Comment:
nit: Generally * imports are frowned upon as they are greedy and can hide subtle bugs.

Reply:
Hi Ross,

Thanks for the feedback! I realized based on this that we can just delete this file from the PR, as we rewrote it and used it in another PR. I also updated the write application in the other PR to remove * imports. The reason not all imports are used is that in the //Start example code here section of the code, users are instructed to copy and paste in code snippets that we have later on in the page; each snippet uses different imports, so while the imports aren't used immediately, they're all used eventually. That's also why we have the //Start example code here section, to indicate where people should copy in the code snippets. The other PR is [here](https://github.com/mongodb/docs-java-rs/pull/38) if you want more context!